testsuite/features/step_definitions/common_steps.rb

```diff
@@ -83,23 +83,25 @@
 # bare metal
 When(/^I check the ram value$/) do
   get_ram_value = "grep MemTotal /proc/meminfo |awk '{print $2}'"
-  ram_value, _local, _remote, _code = $client.test_and_store_results_together(get_ram_value, 'root', 600)
+  ram_value, _local, _remote, _code = $client.run(get_ram_value, 'root', 600)
```

Comment:
ok, thanks - but beware the order of parameters for `run`:
```
  def run(cmd, fatal = true, timeout = DEFAULT_TIMEOUT, user = 'root')
```
also, `root` being optional, it can be omitted.